components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowRequestDAO.java

```diff
@@ -126,6 +126,54 @@ public WorkflowRequest retrieveWorkflow(String uuid) throws InternalWorkflowExce
         return null;
     }
 
+    public static String retrieveExternalWorkflowId(String workflowId) throws InternalWorkflowException {
```

Comment:
Remove the static modifier and add method comments. You can refer another method in this class.